ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -37,6 +39,15 @@ node('caasp-team-private-integration') {
                 platform = pr_platform_label.name.split(":")[1]
            }
 
+           // check if the PR request an specific test platform
+           def pr_experimental_label = pr.labels.find {
+               it.name.startsWith("ci-experimental")
+           }
+           if (pr_experimental_label != null) {
+               worker_type = 'experimental'
+           }
+
+           //check if the PR requires an experimental feature
```

Comment:
This needs to be switched with the above comment.

Reply:
good catch, thanks.